fix: LSP diagnostic icons

deneiruy opened the pull request:
LSP diagnostic icons don't show correctly in the gutter. Line numbers are also not colored correctly:
![wrong symbols](https://user-images.githubusercontent.com/10150931/144988780-8f832b02-7c62-443b-b2a5-3d9045dffd24.png)

Fixed (reference: [nvim-lspconfig README](https://github.com/neovim/nvim-lspconfig/wiki/UI-customization#change-diagnostic-symbols-in-the-sign-column-gutter)):
![fixed](https://user-images.githubusercontent.com/10150931/144988942-8dfc910c-c5d2-47fb-a0ac-a53bbbcfdd5e.png)

deneiruy commented:
> is this on 0.6?

You mean neovim version right? Yep, 0.6
![Screen Shot 2021-12-07 at 4 00 02 PM](https://user-images.githubusercontent.com/10150931/144989542-3d1d5c45-6ddf-4388-971f-67560eb83a06.png)

siduck commented:
> > is this on 0.6?
> 
> You mean neovim version right? Yep, 0.6 ![Screen Shot 2021-12-07 at 4 00 02 PM](https://user-images.githubusercontent.com/10150931/144989542-3d1d5c45-6ddf-4388-971f-67560eb83a06.png)

oh then move this PR into the dev branch!

deneiruy commented:
Oops, didn't know 0.6 had to be on the `dev` branch. The [CONTRIBUTING.md](https://github.com/NvChad/NvChad/blob/main/.github/CONTRIBUTING.md) doc didn't mention it. Let me fix the PR in a bit

siduck commented:
btw why is the nvimtree conf changed?
https://github.com/NvChad/NvChad/pull/630/files#diff-a40ad61478516875d350f672c80109d49ba2e65e3447e3ac137e655c005edfd1L19

deneiruy commented:
> btw why is the nvimtree conf changed? https://github.com/NvChad/NvChad/pull/630/files#diff-a40ad61478516875d350f672c80109d49ba2e65e3447e3ac137e655c005edfd1L19

Sorry about that. It was caused when I changed the target branch to `dev`. 

I've updated the PR. Please do check. Thanks!
